packages/acceptance-tests/pkg-tests-specs/sources/pnpapi.test.ts

```diff
@@ -49,8 +45,8 @@ describe(`Plug'n'Play API`, () => {
           [`no-deps`]: `1.0.0`,
         },
       }, async ({path, run, source}) => {
-        await xfs.mkdirpPromise(`${path}/workspace`);
-        await xfs.writeJsonPromise(`${path}/workspace/package.json`, {
+        await xfs.mkdirpPromise(`${path}/workspace` as PortablePath);
+        await xfs.writeJsonPromise(`${path}/workspace/package.json as PortablePath` as PortablePath, {
```

Comment:
![](https://media0.giphy.com/media/13NRvWtOiMXawM/giphy.gif)

Reply:
😄 Thanks for fixing!